components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -106,6 +108,12 @@
     <string name="mozac_feature_prompts_expand_credit_cards_content_description">Expand suggested credit cards</string>
     <!-- Content description for collapsing the select credit card options in the select credit prompt. -->
     <string name="mozac_feature_prompts_collapse_credit_cards_content_description">Collapse suggested credit cards</string>
-    <!-- Option in the expanded select credit card prompt that links to credit cards settings -->
+    <!-- Option in the expanded select credit card prompt that links to credit cards settings. -->
     <string name="mozac_feature_prompts_manage_credit_cards">Manage credit cards</string>
+    <!-- Text for the title of a save credit card dialog. -->
+    <string name="mozac_feature_prompts_save_credit_card_prompt_title">Securely save this card?</string>
+    <!-- Text for the title of an update credit card dialog. -->
+    <string name="mozac_feature_prompts_update_credit_card_prompt_title">Update card expiration date?</string>
+    <!-- Subtitle text displayed under the title of the save credit card dialog. -->
+    <string name="mozac_feature_prompts_save_credit_card_prompt_body">Card number will be encrypted. Security coded won\'t be saved.</string>
```

Comment:
@gabrielluong 

Could you update this string to use a proper apostrophe, instead of escaping a straight quote? That should be the guideline for Mozilla products.

```
Card number will be encrypted. Security coded won’t be saved.
```

Reply:
Also, now that I notice it, I'm pretty sure there's a typo (coded -> codes)

Reply:
https://github.com/mozilla-mobile/android-components/issues/12145